src/main/java/space/obminyashka/items_exchange/service/impl/SendGridService.java

```diff
@@ -51,7 +55,14 @@ public void sendMail(String emailTo, EmailType subject, Locale locale) throws IO
 
     @Override
     public void validateEmail(UUID validationCode) throws EmailValidationCodeNotFoundException, EmailValidationCodeExpiredException {
```

Comment:
Can you count how many SQL requests would produce invoking of that method?

Reply:
@Sershok the question is still opened

Reply:
updated